main.tf

```diff
@@ -104,6 +104,26 @@ resource "google_compute_instance_from_template" "this" {
   source_instance_template = module.instance_template.self_link
 }
 
+resource "null_resource" "openvpn_install_script" {
```

Comment:
Add a little comment above this to say what this part does.

Reply:
Okay I have done these changes locally

Reply:
Have pushed the changes